src/components/ProductGrid.jsx

```diff
@@ -0,0 +1,17 @@
+import ProductCard from "./ProductCard";
+
+export default function ProductGrid(){
+    return(
+        <main>
+            <div class="product-grid">
+                <ProductCard />
```

Comment:
Suggestion: To avoid the repetition of `<ProductCard` />, you may use Array map() method. rest of the codes look good to me.

Reply:
Yess will use map once I setup mockbee